src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -32,27 +33,32 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
     private _stripeElement?: StripeElement;
     private _stripeCardElements?: StripeCardElements;
     private _useIndividualCardFields?: boolean;
-
+    private _hostedForm?: HostedForm;
+    private _shouldRenderHostedForm?: boolean;
     constructor(
         private _store: CheckoutStore,
         private _paymentMethodActionCreator: PaymentMethodActionCreator,
         private _paymentActionCreator: PaymentActionCreator,
         private _orderActionCreator: OrderActionCreator,
         private _stripeScriptLoader: StripeV3ScriptLoader,
         private _storeCreditActionCreator: StoreCreditActionCreator,
+        private _hostedFormFactory: HostedFormFactory,
         private _locale: string
     ) {}
 
     async initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
         this._initializeOptions = options;
-
         const paymentMethod = this._store.getState().paymentMethods.getPaymentMethodOrThrow(this._getInitializeOptions().methodId);
         const { initializationData: { stripePublishableKey, stripeConnectedAccount, useIndividualCardFields } } = paymentMethod;
 
         this._useIndividualCardFields = useIndividualCardFields;
         this._stripeV3Client = await this._loadStripeJs(stripePublishableKey, stripeConnectedAccount);
         this._stripeElement = await this._mountElement(this._getInitializeOptions().methodId);
 
+        if (StripePaymentMethodType.CreditCard && this._isHostedPaymentFormEnabled(this._getInitializeOptions().methodId, this._getInitializeOptions().gatewayId) && this._isHostedFieldAvailable() && options.stripev3 && options.stripev3.form) {
```

Comment:
this line is quite long, I would recommend splitting each condition into a different line for readabality.